opensubtitles/download.go

```diff
@@ -0,0 +1,62 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+// DownloadService provides access to the download related functions
+// in the OpenSubtitles API.
+//
+// OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#download
+type DownloadService service
+
+type DownloadOptions struct {
```

Comment:
Codacy found an issue: [exported type DownloadOptions should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)